stdlib/Test/test/runtests.jl

```diff
@@ -1712,13 +1712,13 @@ end
 
         # this tests both the `TestCounts` parts as well as the fallback `x`s
         expected = r"""
-                    Test Summary: | Pass  Fail  Error  Broken  Total  Time
-                    outer         |    3     1      1       1      6  \s*\d*.\ds
-                      a           |    1                           1  \s*\d*.\ds
-                      custom      |    1     1      1       1      4  \s*?s
-                      no-record   |    x     x      x       x      ?  \s*?s
-                      b           |    1                           1  \s*\d*.\ds
-                    ERROR: Some tests did not pass: 3 passed, 1 failed, 1 errored, 1 broken.
+                    Test Summary: \| Pass  Fail  Error  Broken  Total  Time
+                    outer         \|    3     1      1       1      6  \s*\d*.\ds
+                      a           \|    1                           1  \s*\d*.\ds
+                      custom      \|    1     1      1       1      4  \s*\?s
+                      no-record   \|    x     x      x       x      \?  \s*\?s
+                      b           \|    1                           1  \s*\d*.\ds
+                    Random seed of the outermost testset: .*
```

Comment:
@Seelengrab this test introduced in #53215 was completely broken because the `|` were interpreted inside the regex as alternatives, so it was sufficient to match any of the alternatives to get the test to pass, writing total garbate (like the `ERROR` line which isn't even in the output, or the `?` symbols which actually have a meaning inside regex unless escaped) would be irrelevant for the purpose of matching.

Reply:
Whoops, thank you! The fix is good though.